lib/commons/forms/is-aria-range.js

```diff
@@ -0,0 +1,14 @@
+/* global forms */
+const rangeRoles = ['progressbar', 'scrollbar', 'slider', 'spinbutton'];
+
+/**
+ * Determines if an element is an aria range element
+ * @method isAriaRange
+ * @memberof axe.commons.forms
+ * @param {Element} node Node to determine if aria range
+ * @returns {Bool}
+ */
+forms.isAriaRange = function(node) {
+	const role = axe.commons.aria.getRole(node, { noImplicit: true });
+	return rangeRoles.includes(role) && node.hasAttribute('aria-valuenow');
```

Comment:
I don't think we should include the `hasAttribute` test here. Even without aria-valuenow, it's still an aria range element. This check is going to make reuse of this function problematic. Better to move the attribute check part outside this function IMO.